### DISCUSSION TITLE
emqx docker auth http respond very slow

### DISCUSSION THREAD

**Discussion by jsw-davidhuang** [emqx/emqx#9700] [Q&A]
i use emqx 4.3 docker image, then my auth server is also in the same machine, for exmaple:
emqx ip: 172.17.0.1
auth server ip:172.17.0.2
when i allow_anonymous and disable the auth_http plugin , i use mqtt client to connect is very fast.
but when i enable the auth_http plugin,  the auth_http request url is http://172.17.0.2/mqtt/auth,  then i use mqtt client to connect is very slow like 8s to connect. but i use curl to do the http post request in 172.17.0.1 is very fast, i am not sure it is a docker issue or emqx issue?

**Comment by lafirest** [comment]
@jsw-davidhuang 
Could you please set the log level to debug and then make some logs, the logs can help us to know what happened clearly

**Reply by jsw-davidhuang** [reply]
no need now, i upgrade to version 5.0.x and the http is real fast now

**Reply by jsw-davidhuang** [reply]
i think you can close this discussion
